src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -362,7 +362,7 @@ private void V3SkipNewlines()
             if (_ungotToken == null || _ungotToken.Kind == TokenKind.NewLine)
             {
                 _ungotToken = null;
-                _tokenizer.SkipNewlines(false, true);
+                _tokenizer.SkipNewlines(NewlineSkipOption.V3);
```

Comment:
Please look NoteV3FeatureUsed() - I think we can remove V3.

Reply:
Yes, this method can be removed. The original intent was to be helpful for a compatibility checker. Features like new keywords would be obvious, but places where PowerShell was more permissive with newlines would have been much harder to detect.

That compatibility checker was never built and nobody cares about V2 to V3 compat anymore.

---
In reply to: [258930501](https://github.com/PowerShell/PowerShell/pull/8938#discussion_r258930501) [](ancestors = 258930501)

Reply:
Agreed. I didn't want to touch more than I needed to, but that logic lead to commented out code that is no longer used, so I just cleaned all of that up. Thank you for suggesting I do so.